backend/src/v5/middleware/dataConverter/outputs/teamspaces/projects/models/commons/tickets.comments.js

```diff
@@ -15,16 +15,36 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const { UUIDToString } = require('../../../../../../../utils/helper/uuids');
+const Yup = require('yup');
+const { logger } = require('../../../../../../../utils/logger');
 const { respond } = require('../../../../../../../utils/responder');
-const { serialiseComment } = require('../../../../../../../schemas/tickets/comments');
 const { templates } = require('../../../../../../../utils/responseCodes');
+const { types } = require('../../../../../../../utils/helper/yup');
 
 const Comments = {};
 
+const uuidString = Yup.string().transform((val, orgVal) => UUIDToString(orgVal));
+
+const serialiseComment = (comment) => {
+	const caster = Yup.object({
+		_id: uuidString,
+		createdAt: types.timestamp,
+		updatedAt: types.timestamp,
+		images: Yup.array().of(uuidString),
+		history: Yup.array().of(Yup.object({
+			timestamp: types.timestamp,
+			images: Yup.array().of(uuidString),
+		})),
+	});
+	return caster.cast(comment);
+};
+
 Comments.serialiseComment = (req, res) => {
 	try {
 		respond(req, res, templates.ok, serialiseComment(req.commentData));
 	} catch (err) {
+		logger.logError(`Failed to serialise comment: ${err}`);
```

Comment:
err is likely to be an object, try for the message first
```suggestion
		logger.logError(`Failed to serialise comment: ${err.message || err}`);
```